### PR TITLE
Selenium issue 8724 - strings are coerced to integers when they shouldn't be

### PR DESCRIPTION
**Pull request by AutomatedTester**

### CODE REVIEW

File: webdriver/tests/get_element_property/get.py [hunk 1]
```diff
@@ -131,3 +131,10 @@ def test_anchor_href(session, inline, url, is_relative):
 
     response = get_element_property(session, element.id, "href")
     assert_success(response, url("/foo.html"))
+
+
+def test_strings_not_coerced_into_integers(session, inline):
+    session.url = inline("<li  value=\"4b273a33fbbd29013nN93dy4F1A~\" class=\"cur\"></li>")
+    element = session.find.css("li", all=False)
+    result = get_element_property(session, element.id, "value")
```
Comment (jgraham) [inline webdriver/tests/get_element_property/get.py:139]: I think this is wrong, per https://software.hixie.ch/utilities/js/live-dom-viewer/?%3C!DOCTYPE%20html%3E%0A%3Cli%20value%3D%224b273a33fbbd29013nN93dy4F1A~%22%3Etest%3C%2Fli%3E%0A%3Cscript%3E%0Aw(document.getElementsByTagName(%22li%22)%5B0%5D.value)%0A%3C%2Fscript%3E the result should be 4 (!).

**Review by jgraham — CHANGES_REQUESTED** [review]

File: webdriver/tests/get_element_property/get.py (hunk 1 above)
Comment (gsnedders) [reply webdriver/tests/get_element_property/get.py:139]: @AutomatedTester ping

Comment (gsnedders) [reply webdriver/tests/get_element_property/get.py:139]: (note that every browser returns "4" (an integer) here, which is per-spec)

**Comment by whimboo** [comment]
@AutomatedTester are you still interested to finish this PR or shall we close it?

**Pull request closed without merging**